[
  {
    "id" : "e8632152-6033-4e30-ad51-5d5fa54cb671",
    "prId" : 32547,
    "prUrl" : null,
    "prSource" : "GitHub",
    "comments" : [
      {
        "id" : "af4b152a-a135-4030-9603-ae49920eeff1",
        "parentId" : null,
        "authorId" : "8be927c4-cfb4-4077-b355-f4f3d84849b8",
        "body" : "made this a package-level helper... should probably move peer to `client/unversioned#RetryOnConflict` if there's broad agreement that a retry on a 5xx makes sense\n",
        "createdAt" : "2016-09-13T02:24:09Z",
        "updatedAt" : "2016-09-13T04:42:12Z",
        "lastEditedBy" : "8be927c4-cfb4-4077-b355-f4f3d84849b8",
        "tags" : [
        ]
      }
    ],
    "commit" : "e9914f2c4af2c41b7682d65378864246a34e2465",
    "line" : 27,
    "diffHunk" : "@@ -1,1 +85,89 @@// WithExponentialBackoff will retry webhookFn() up to 5 times with exponentially increasing backoff when\n// it returns an error for which apierrors.SuggestsClientDelay() or apierrors.IsInternalError() returns true.\nfunc WithExponentialBackoff(initialBackoff time.Duration, webhookFn func() error) error {\n\tbackoff := wait.Backoff{\n\t\tDuration: initialBackoff,"
  }
]